packages/cli/src/util/build/write-build-result.ts

```diff
@@ -162,19 +183,31 @@ async function writeBuildResultV2(
 async function writeBuildResultV3(
   outputDir: string,
   buildResult: BuildResultV3,
-  build: Builder
+  build: Builder,
+  vercelConfig: VercelConfig | null
 ) {
   const { output } = buildResult;
   const src = build.src;
   if (typeof src !== 'string') {
     throw new Error(`Expected "build.src" to be a string`);
   }
+
+  let customFunctionConfiguration: FunctionConfiguration | undefined;
+  if (vercelConfig?.functions) {
```

Comment:
Can we re-use this function? `getLambdaOptionsFromFunction()` from build utils?

Reply:
6d13ed54c551c20307f4610a5359e55bef59e8e0